tests/v1/voting_information/test_postcode.py

```diff
@@ -46,11 +48,18 @@ def test_valid(
     with patch(
         "api.endpoints.v1.voting_information.address.DCWidePostcodeLoggingClient.log"
     ) as mock_log:
-        response = vi_app_client.get(f"/api/v1/postcode/{postcode}/")
+        url = f"/api/v1/postcode/{postcode}"
+
+        if postcode == "CC12CC":
+            url += "?include_accessibility=true"
```

Comment:
I had to do this because of the way I implemented the sandbox response. The problem is that this case isn't just a postcode it is also a parameter.

I did consider just skipping this postcode here, but I think having this test in place (the intention of it is to ensure we don't make schema changes without updating the sandbox responses) for all the sandbox responses is useful, and worth a bit of messyness in the test code.

I don't love this though.